### ISSUE TITLE
show index statement return a wrong result

### ISSUE THREAD

**Issue by Qubutol** [vesoft-inc/nebula#1952]
**Describe the bug (__must be provided__)**

usage [show tag indexes] statement,but return include both tag and edge index

**Your Environments (__must be provided__)**

OS:centos7 x86_64
version:1.0.0-rc3

**How To Reproduce(__must be provided__)**

1. Step 1
    CREATE TAG INDEX player_index_0 on player(name);
    CREATE EDGE INDEX follow_index_0 on follow(degree);
2. Step 2
    show tag indexes
3. Step 3
```
=================
| Index ID | Index Name  |
=================
| 14       | player_index_0 |
-----------------------------
| 15       | follow_index_0 |
-----------------------------
```
4. Step 4
    show edge indexes
5.Step 5
```
=================
| Index ID | Index Name  |
=================
| 14       | player_index_0 |
-----------------------------
| 15       | follow_index_0 |
-----------------------------
```

**Expected behavior**

only return tag or edge index

**Comment by Shinji-IkariG** [comment]
After testing, this problem does exist in rc3.0, this problem has been fixed in rc4.0, thank you for your feedback.

**Comment by whitewum** [comment]
> .0, this problem has been fixe

which pr? please ref to this issue. So we can close it.

**Comment by bright-starry-sky** [comment]
This issue fixed by PR #1675 , close this issue.
